python/tutorials/fused-attention-ws-device-tma.py

```diff
@@ -141,7 +141,12 @@ def _attn_fwd_inner_oss_dp(
 
     # loop over k, v and update accumulator
     for start_n in tl.range(
-        lo, hi, BLOCK_N, warp_specialize=warp_specialize, disallow_acc_multi_buffer=True, data_partition_factor=DP_FACTOR,
+            lo,
```

Comment:
In general I would avoid unnecessary changes like this.

Reply:
Thanks for catching them all! I removed this change too